docs/site/api.md

```diff
@@ -157,6 +157,8 @@ Example: [/api/v2/contest/bts19](https://dmoj.ca/api/v2/contest/bts19)
     "rankings": [
         {
             "user": "<participant username>",
+            "start_time": "<participation start time in ISO format>",
```

Comment:
Maybe `"effective participation start time..."` is more accurate? The current description is more fit for `real_start`.